### ISSUE TITLE
Test suite fails on GHC 8.8.4

### ISSUE THREAD

**Issue by dminuoso** [byteverse/bytebuild#19]
Building test suite 'test' for bytebuild-0.3.6.0..
This currently marks bytebuild as broken on nixpkgs
```
[1 of 3] Compiling HexWord64        ( common/HexWord64.hs, dist/build/test/test-tmp/HexWord64.o )
[2 of 3] Compiling Word16Tree       ( common/Word16Tree.hs, dist/build/test/test-tmp/Word16Tree.o )
[3 of 3] Compiling Main             ( test/Main.hs, dist/build/test/test-tmp/Main.o )

test/Main.hs:324:20: error:
    Ambiguous occurrence ‘replicate’
    It could refer to
       either ‘Prelude.replicate’,
              imported from ‘Prelude’ at test/Main.hs:1:1
              (and originally defined in ‘GHC.List’)
           or ‘Data.Bytes.Builder.replicate’,
              imported from ‘Data.Bytes.Builder’ at test/Main.hs:10:1-25
    |
324 |   | length s < n = replicate (n - length s) '0' ++ s
    |                    ^^^^^^^^^
```

**Comment by andrewthad** [comment]
Thanks for the issue. I've resolved this in 04ab7f046b7bb21453a80249f22cf1d907aa5f7a, which will be released to hackage on a later date.

**Comment by andrewthad** [comment]
This made it to hackage in 0.3.11.0. Closing.
